core-api/src/main/java/com/optimizely/ab/bucketing/DecisionService.java

```diff
@@ -307,7 +307,10 @@ public DecisionService(@Nonnull Bucketer bucketer,
 
         int index = 0;
         while (index < rolloutRulesLength) {
-
+            if (!rollout.getExperiments().get(index).isRunning()) {
```

Comment:
Should this check the status of the experiment to be "paused" rather than is running since there are 5 other status'? Or is this behavior supposed to be for every status other than running?